packages/shared/src/components/MainLayout.tsx

```diff
@@ -81,11 +80,6 @@ function MainLayout({
   useAuthVerificationRecovery();
   useNotificationParams();
   useReferralReminder();
-  const handlers = useSwipeableSidebar({
```

Comment:
Nimrod asked to remove completely the swipes for now

Reply:
Let's see how long before people start complaining 🤣